libs/sdk-ui-ext/src/internal/components/configurationControls/interactions/InteractionsSection.tsx

```diff
@@ -7,9 +7,11 @@ import { messages } from "../../../../locales.js";
 import { IVisualizationProperties } from "../../../interfaces/Visualization.js";
 import CheckboxControl from "../CheckboxControl.js";
 import { Bubble, BubbleHoverTrigger, SeparatorLine } from "@gooddata/sdk-ui-kit";
+import { SectionName } from "../../configurationPanels/sectionName.js";
 
 export interface IInteractionsSectionProps {
-    controlsDisabled: boolean;
+    controlsDisabled?: boolean;
+    isControlsDisabled?: (sectionName?: SectionName) => boolean;
```

Comment:
I find the naming unfortunate here. Maybe something like
```suggestion
    areControlsDisabled?: boolean;
    areSectionControlsDisabled?: (sectionName?: SectionName) => boolean;
```

Reply:
original flag kept as was as it is spread everywhere. New getter renamed